libs/log.c

```diff
@@ -57,6 +58,12 @@ log_open(void)
 		expanded_path = expand_path(path);
 
 		path = fxstrdup(expanded_path);
```

Comment:
Not sure why this line is here, it should have been deleted (and only used if starts with /).

```suggestion
```

Reply:
Better, but we need to do a bit more, as in:

```
diff --git a/libs/log.c b/libs/log.c
index 23a65a70c..977c13c51 100644
--- a/libs/log.c
+++ b/libs/log.c
@@ -47,23 +47,21 @@ log_set_level(int ll)
 void
 log_open(const char *fvwm_userdir)
 {
-       char *path;
+       char *path, *logfile_env;
+
        xasprintf(&path, "%s/%s", fvwm_userdir, FVWM3_LOGFILE_DEFAULT);
 
-       if (getenv("FVWM3_LOGFILE")) {
+       logfile_env = getenv("FVWM3_LOGFILE");
+       if (logfile_env != NULL) {
                const char      *expanded_path;
 
                free(path);
-               path = getenv("FVWM3_LOGFILE");
-               expanded_path = expand_path(path);
+               expanded_path = expand_path(logfile_env);
 
-               path = fxstrdup(expanded_path);
                if (expanded_path[0] == '/')
-               {
-                               path = fxstrdup(expanded_path);
-               } else {
-                               xasprintf(&path, "%s/%s", fvwm_userdir, expanded_path);
-               }
+                       path = fxstrdup(expanded_path);
+               else
+                       xasprintf(&path, "%s/%s", fvwm_userdir, expanded_path);
                free((char *)expanded_path);
        }
```

Reply:
Additionally, this code change in its entirety needs to be split off as a separate commit, and not just shoe-horned in to the other commits as it's nothing to do with these on-going documentation changes.

@d-e-e-p -- are you OK doing that?